lib/main.dart

```diff
@@ -31,5 +33,15 @@ void main() async {
     return true;
   };
 
+  HttpOverrides.global = MyHttpOverrides();
   runApp(App());
 }
+
+class MyHttpOverrides extends HttpOverrides {
+  @override
+  HttpClient createHttpClient(SecurityContext? context) {
+    return super.createHttpClient(context)
+      ..badCertificateCallback =
+          (X509Certificate cert, String host, int port) => true;
+  }
+}
```

Comment:
Hmm 🤔 can you add an explication about that?

Is it temporary, can't it be done in Dio, the SO answer where you find that snippet, stuff like that